<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExtensionManager</class>
 <widget class="QDialog" name="FileExtensionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonUp">
       <property name="text">
        <string>&amp;Up</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/up</normaloff>:/icons/up</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDown">
       <property name="text">
        <string>&amp;Down</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/down</normaloff>:/icons/down</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/field_add</normaloff>:/icons/field_add</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/field_delete</normaloff>:/icons/field_delete</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableExtensions">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Extensions</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileExtensionManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileExtensionManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
